include/kernel/boot.h

```diff
@@ -135,3 +135,10 @@ static inline BOOT_CODE pptr_t it_alloc_paging(void)
 
 /* return the amount of paging structures required to cover v_reg */
 word_t arch_get_n_paging(v_region_t it_veg);
+
+#if defined(CONFIG_DEBUG_BUILD) && defined(ENABLE_SMP_SUPPORT) && defined(CONFIG_KERNEL_MCS)
+/* Test whether clocks are synchronised across nodes */
+BOOT_CODE void clock_sync_test(void);
+#else
+#define clock_sync_test()
```

Comment:
@lsf37: can't we make this #define clock_sync_test() ((void)(0)) (or do {} while false) because the proofs can handle this as a no-op? Then we'd have statement here that requires/consume a ; and looks like a function